src/api/migraphx.py

```diff
@@ -308,6 +314,16 @@ def operation(h):
                  fname='migraphx::save')
 
 
+@api.handle('migraphx_execution_environment',
+            'migraphx::execution_environment')
+def execution_environment(h):
```

Comment:
I dont think we should expose the `execution_environment` class.

Reply:
removed